Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Driver/Registration.hs

```diff
@@ -51,19 +59,36 @@ import Kernel.Utils.Common
 import qualified SharedLogic.DriverOnboarding as DomainRC
 import SharedLogic.Merchant (findMerchantByShortId)
 import qualified Storage.CachedQueries.Merchant.MerchantOperatingCity as CQMOC
+import qualified Storage.Queries.DriverLicense as QDL
+import qualified Storage.Queries.DriverSSN as QSSN
 import qualified Storage.Queries.FleetDriverAssociation as QFDV
 import Storage.Queries.Image as QImage
+import qualified Storage.Queries.Person as QDriver
+import qualified Storage.Queries.VehicleFitnessCertificate as QFC
+import qualified Storage.Queries.VehicleInsurance as QVI
+import qualified Storage.Queries.VehicleRegistrationCertificate as QRC
 import qualified Tools.AadhaarVerification as AadhaarVerification
+import Tools.Error
+import Tools.Notifications as Notify
 
 documentsList :: ShortId DM.Merchant -> Context.City -> Id Common.Driver -> Flow Common.DocumentsListResponse
 documentsList merchantShortId _ driverId = do
   merchant <- findMerchantByShortId merchantShortId
   licImgs <- map (.id.getId) <$> runInReplica (findImagesByPersonAndType merchant.id (cast driverId) Domain.DriverLicense)
   vehRegImgs <- map (.id.getId) <$> runInReplica (findImagesByPersonAndType merchant.id (cast driverId) Domain.VehicleRegistrationCertificate)
+  uploadProfImgs <- map (.id.getId) <$> runInReplica (findImagesByPersonAndType merchant.id (cast driverId) Domain.UploadProfile)
+  vehicleFitnessCertImgs <- map (.id.getId) <$> runInReplica (findImagesByPersonAndType merchant.id (cast driverId) Domain.VehicleFitnessCertificate)
+  vehicleInsImgs <- map (.id.getId) <$> runInReplica (findImagesByPersonAndType merchant.id (cast driverId) Domain.VehicleInsurance)
+  ssnEntry <- QSSN.findByDriverId (cast driverId) >>= fromMaybeM (InternalError $ "SSN not found for driverid: " <> driverId.getId)
```

Comment:
Lets not throw error here and return empty. Else this API will break in production for Bangalore drivers.

Reply:
Yes, have correct it here https://github.com/nammayatri/nammayatri/commit/9319ad66200de0761f5fd3e138a76f175f1d5d26
Will get it merged and add other fixed too.